global.json

```diff
@@ -1,7 +1,7 @@
 {
   // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
     "msbuild-sdks": {
-        "Uno.Sdk.Private": "6.1.0-dev.667",
```

Comment:
@agneszitte @kazo0 we can switch to public api for main at least?
probably need to make another edit for canary after merging this, or we drop uno.sdk.private completely?